fiftyone/utils/openimages.py

```diff
@@ -984,7 +1087,7 @@ def _get_all_label_data(
     all_attrs_ids = set(image_ids)
     any_attrs_ids = set()
 
-    _label_types = _parse_label_types(label_types)
+    _label_types = label_types
```

Comment:
As is, it works as you would expect/desire. When `label_types=None`, it assumes all label types that are allowed for that version should be included.

Reply:
When `label_types=None`, this will raise an error when it gets to `if "classifications" in _label_types`. Following my other comments, I'd suggest updating the implementation so `label_types=None` is still allowed by adding a required `version` argument to this method.